Black-Market/Black-Market/DemoComponentsView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ContentView.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 7/10/23.
+//
+
+import SwiftUI
+import UIKit
+
+struct DemoComponentsView: View {
+    @State var input = ""
+    var body: some View {
+        VStack(spacing: 15) {
+            Text("My components")
```

Comment:
I understand this is a demo, but please remember to localize all the strings for better scalability.

Reply:
I got it, In the next PR I am going to put the strings in localizable file.